docs/tests/visdiff.js

```diff
@@ -169,14 +169,21 @@ describe('docs site', () => {
 // - page is a reference to the Puppeteer page.
 // - route is the path you're loading, which I'm using to name the file.
 // - filePrefix is either "wide" or "narrow", since I'm automatically testing both.
-async function takeAndCompareScreenshot(page, route, filePrefix) {
+async function takeAndCompareScreenshot(page, route, filePrefix, delay = 0) {
   // If you didn't specify a file, use the name of the route.
   let fileName = filePrefix + '/' + (route ? route : 'index');
 
   // Start the browser, go to that page, and take a screenshot.
   await page.goto(`http://localhost:8080/${route}`, {
     waitUntil: 'load'
   });
+  // delay, to make sure everything has been rendered and stable (retry would be better...)
+  if (delay) {
```

Comment:
page load itself is a function that every thing is loaded, why the extra delay at all?

Reply:
There may be changes to the page after load, I had to deal with this myself, when trying to implement foxr. On all the component pages there are animations, which can cause failures.

Reply:
correct. I found during local tests the images in card are not always loaded, although page.load should be the correct hook. Looks like loaded does not (always) mean rendered as well...
Localy the only other problem was the LinearProgress because of the animation. Later in Travis it was the Textfield failing test, but only in NodeJS6, not in NodeJS8. So this looks like timing issue too...

Reply:
That's the reason why I would like to get away from screenshot testing in the future, but that's another topic and definitely `2.0` or later